concent_api/core/tests/test_payments_bankster.py

```diff
@@ -241,7 +241,7 @@ def test_that_when_there_are_other_deposit_claims_finalize_payment_substract_the
 
 class SettleOverdueAcceptancesBanksterTest(ConcentIntegrationTestCase):
 
-    def test_that_settle_overdue_acceptances_should_return_empty_claim_deposit_info_if_subtask_costs_where_already_paid(self):
+    def test_that_settle_overdue_acceptances_should_return_empty_claim_deposit_if_subtask_costs_where_already_paid(self):
```

Comment:
it doesn't return empty claim deposit, but a string (with error message)

Reply:
:+1: done